testing.py

```diff
@@ -57,7 +57,7 @@
     WorkCoverageProvider,
 )
 
-from external_search import DummyExternalSearchIndex
+from external_search import MockExternalSearchIndex
```

Comment:
We use 'Mock' elsewhere in the application for this type of class, and this branch is a good time to change the usage here.